datalad/consts.py

```diff
@@ -35,8 +35,6 @@
 ANNEX_TEMP_DIR = join('.git', 'annex', 'tmp')
 ANNEX_TRANSFER_DIR = join('.git', 'annex', 'transfer')
 
-SEARCH_INDEX_DOTGITDIR = join('datalad', 'search_index')
-
```

Comment:
I would leave it in (no cost) and add a comment that it is deprecated/ignored.

Reply:
Thx. Suggestion applied